socorro/external/boto/connection_context.py

```diff
@@ -112,13 +182,11 @@ def _get_credentials(self):
         raise NotImplementedError
 
     #--------------------------------------------------------------------------
-    @staticmethod
-    def build_key(prefix, name_of_thing, id, version='v1'):
```

Comment:
Would it be value in keeping this? It could easily be:
```python
def build_key(self, prefix, name_of_thing, id, version='v1'):
    return [self.keybuilder.build_keys(prefix, name_of_thing, id, version=version)]
```
Note, if you think this is a good idea, we'd need to have `version='v2'` in the new fancy `build_keys()` method.